src/Emulator/Peripherals/Peripherals/Timers/STM32_Timer.cs

```diff
@@ -85,7 +85,7 @@ public STM32_Timer(Machine machine, long frequency, uint initialLimit) : base(ma
                     .WithReservedBits(1, 1)
                     .WithTaggedFlag("CCUS", 2)
                     .WithTaggedFlag("CCDS", 3)
-                    .WithTag("MMS", 4, 2)
+                    .WithFlags(4, 3, name: "MMS")
```

Comment:
In general, we use `Tag` and `TaggedFlag` when we want to explicitly say that we did not implement the relevant logic behind a field. Changing it to `Flag` makes it an r/w field without any real logic behind. This is sometimes ok, but if there **should** be logic - we hide the fact that it's not implemented.

Reply:
well, now it's understandable thanks for explanation, I think it could be good to describe the differences in you documentation because I didn't get why you use Tags instead of flags after reading docs